kernel/acpi/src/lib.rs

```diff
@@ -106,6 +92,7 @@ pub fn init(rsdp_address: Option<PhysicalAddress>, page_table: &mut PageTable) -
             );
 
             for table in dmar_table.iter() {
+                #[allow(clippy::single_match)]
```

Comment:
might as well change it to the `if let` pattern (similar to what you did in the other file)

Reply:
> in acpi lib.rs, where instead of using if let like clippy wanted I'm instead printing that the case is unhandled

I left it like this because I thought a future dev will handle the other variants

Reply:
Resolved in the main branch already.